File: Freedom/微信/发现/WXShoppingViewController.m
//  TLShoppingViewController.m
//  Freedom
// Created by Super
#import "WXShoppingViewController.h"
@implementation WXShoppingViewController
- (void) viewDidLoad{
    [super viewDidLoad];
    
    UIBarButtonItem *rightBarButton = [[UIBarButtonItem alloc] initWithImage:[UIImage imageNamed:@"nav_shopping_menu"] style:UIBarButtonItemStylePlain target:self action:@selector(rightBarButtonDown:)];
    [self.navigationItem setRightBarButtonItem:rightBarButton];
    
//    [self setUrl:@"http://wq.jd.com/"];
    [self setUrl:@"http://m.zhuanzhuan.com"];
}
#pragma mark - Event Response
- (void) rightBarButtonDown:(UIBarButtonItem *)sender{
}
@end

File: Freedom/微信/发现/WXShoppingViewController.swift
//
//  WXShoppingViewController.swift
//  Freedom

import Foundation
class WXShoppingViewController: WXWebViewController {
    override func viewDidLoad() {
        super.viewDidLoad()
        let rightBarButton = UIBarButtonItem(image:Image.shopping.image, style: .plain, target: self, action: #selector(WXShoppingViewController.rightBarButtonDown(_:)))
        navigationItem.rightBarButtonItem = rightBarButton
        self.url = "http://m.zhuanzhuan.com"
    }
    @objc func rightBarButtonDown(_ sender: UIBarButtonItem) {
    }
}
